resources/telemetry/values.yaml

```diff
@@ -13,7 +13,7 @@ global:
       version: "PR-16573"
     telemetry_otel_collector:
       name: "otel-collector"
-      version: "0.66.0-a80d981f"
+      version: "0.66.0-8bb1c644"
```

Comment:
Maybe change the default image [here](https://github.com/kyma-project/kyma/blob/main/components/telemetry-operator/main.go#L101). That makes testing a bit easier.

Also update the telemetry_operator image to `PR-16558` to run integration tests.

Reply:
image updated